wrapper/CSharp/wolfSSL-TLS-Client/wolfSSL-TLS-Client.cs

```diff
@@ -29,16 +38,70 @@
 
 public class wolfSSL_TLS_Client
 {
+    // Sample listening server:
+    // See https://github.com/wolfSSL/wolfssl/tree/master/examples/server
+    //
+    // Examples disable client cert check with `-d`:
+    //   ./examples/server/server -d -p 11111 -c ./certs/server-cert.pem -k ./certs/server-key.pem
+    //   ./examples/server/server -d -p 12345
+    //
+    // Ensure the -p [port] for client matches SERVER_PORT value here:
+    //
+    public static string SERVER_NAME = "localhost"; /* or IP address: "192.168.1.73 */
+    public static int SERVER_PORT = 11111;
+
+    // Optionally set explicit cipher, see wolfssl.CTX_set_cipher_list()
+    public static string CIPHER_SUITE = "ECDHE-ECDSA-AES128-GCM-SHA256";
+
+    private static WOLFSSL_ALERT_HISTORY myHistory = new WOLFSSL_ALERT_HISTORY();
+
+
+#if WindowsCE
+    /// <summary>
+    /// Example of a logging function for Windows CE, string msg
+    /// </summary>
+    /// <param name="lvl">level of log</param>
+    /// <param name="msg">message to log</param>
+    public static void standard_log(int lvl, string msg)
+    {
+        string logMsg = wolfssl.MultiByteToWideChar(msg);
+        Console.WriteLine(logMsg);
+    }
+#else
     /// <summary>
-    /// Example of a logging function
+    /// Example of a logging function, StringBuilder msg
     /// </summary>
     /// <param name="lvl">level of log</param>
     /// <param name="msg">message to log</param>
     public static void standard_log(int lvl, StringBuilder msg)
     {
         Console.WriteLine(msg);
     }
+#endif
+
+    public static void show_alert_history_code(WOLFSSL_ALERT h, string m)
+    {
+        /* VS initializes .code and .level to zero; wolfSSL sets to -1 until there's a valid value. */
+        if ((h.code > 0) || (h.level > 0)) {

```

Comment:
Please cleanup example. You have `if ((h.code > 0) || (h.level > 0))` twice.

Reply:
Good catch, changed.